### ISSUE TITLE
Hyprland Crashes on single finger touch and edge swipe

### ISSUE THREAD

**Issue by xVermillionx** [horriblename/hyprgrass#47]
I tried the gestures and sometimes Hyprland crashes, but Hyprland crashes every time if its just a single touch interaction (aka touching something on screen), also happens on edge swipe.

Here I have attached the crash log, from just using the touch screen normally (single finger touch), if that helps: 
[hyprlandCrashReport9655.txt](https://github.com/horriblename/hyprgrass/files/12840203/hyprlandCrashReport9655.txt)

Config:
```
plugin {
    touch_gestures {
      sensitivity = 4.0
      workspace_swipe_fingers = 4
    }
}

bind =,edge:d:u,exec,/usr/bin/kill -s 34 $(pidof wvkbd-mobintl) #crashes
bind =,swipe:3:u,exec,/usr/bin/notify-send hi                   #works
bind =,edge:u:d,exec,$term --class dropdownterm                 #crashes
```

I have not looked into this in any detail and won't be able to look at it, but I wanted to report this here (maybe others have the same experience).

Also thank you for creating this plugin to bringing gesture support to Hyprland.

**Comment by horriblename** [comment]
possibly a mismatched version, rebuild hyprgrass.

If that doesn't work, check if you have `/usr/local/include/hyprland`, if so, remove remove `/usr/include/hyprland`, `/usr/local/include/hyprland`, `/usr/share/pkgconfig/hyprland.pc` and `/usr/local/share/pkgconfig/hyprland.pc`, then reinstall Hyprland and hyprgrass

**Comment by xVermillionx** [comment]
Has not worked for me yet, but I will keep trying.

It's only edge swipe and normal touch interactions.
Workspace swipe and multi finger gestures do work perfectly.

I will test it this WE on another machine just in case.

**Comment by crimist** [comment]
hyprland commit: `5b8cfdf`
hyprgrass commit: 878bcda

In both debug and release builds:
* Single touch inputs can crash hyprland
* Log message `[wlr] [types/seat/wlr_seat_touch.c:190] could not create touch point` appears during a single touch input if it does not crash hyprland.
* Bound gestures (swipe etc.) do not crash hyprland

However the crash frequency between debug and release builds is different.
* Release build: single touch input crashes hyprland 9/10 times
* Debug build: single touch input crashes hyprland 1/20 times

Debug crash log: 
[hyprlandCrashReport587496.txt](https://github.com/horriblename/hyprgrass/files/12981148/hyprlandCrashReport587496.txt)

**Comment by crimist** [comment]
[hyprlandd.log](https://github.com/horriblename/hyprgrass/files/12981473/hyprlandd.log)

**Comment by Magnitudexx** [comment]
This happens to me as well

**Comment by crimist** [comment]
Issue has been resolved for me with PR #60. See https://github.com/horriblename/hyprgrass/issues/45#issuecomment-1811624812 for more information.

**Comment by horriblename** [comment]
it seems the problem is fixed for most people, feel free to reopen if not
